assets/js/vue-apps/components/facet-select.vue

```diff
@@ -0,0 +1,34 @@
+<script>
+export default {
+    props: ['value', 'name', 'label', 'labelAny', 'options']
+};
+</script>
+
+<template>
+    <div class="u-margin-bottom-s" v-if="options.length > 0">
+        <label class="ff-label" :for="'field-dynamic-' + name">
+            {{ label }}
+        </label>
+        <select
+            class="ff-select"
+            :name="name" :id="'field-dynamic-' + name"
+            :value="value"
+            @input="$emit('input', $event.target.value)">
+            <option value="">{{ labelAny }}</option>
+            <option
+                v-for="option in options"
+                v-bind:key="option.label"
+                :value="option.value">
+                {{ option.label }} ({{ option.count }})
```

Comment:
Could we use `.toLocaleString()` on these counts (eg. to turn `24343` into `24,343`) for readability? Although you'd have to write a function as you can't do it inline (eg. `123.toLocaleString()` fails, but `let n = 123; n.toLocaleString()` works).